templates/tutorial/tutorial/diff.html

```diff
@@ -38,25 +38,34 @@ <h1 {% if tutorial.image %}class="illu"{% endif %}>
 
 
 {% block content_ext %}
-    <h2>{% trans "Nouveaux Fichiers" %}</h2>
-    {% for add in path_add %}
-        {% with add_next=add.b_blob|repo_blob %}
-            <h3>{{ add.a_blob.path }}</h3>
-            <div class="diff_delta">
-                {{ ''|diff_text:add_next|safe }}
-            </div>
-        {% endwith %}
+    <h2>{% trans "Fichiers modifiés" %}</h2>
+    {% for diff in modified %}
+        <h3>{{ diff.a_path }}</h3>
+        {% htmldiff diff.a_blob.data_stream.read diff.b_blob.data_stream.read %}
+        {% empty %}
```

Comment:
Un niveau d'indentation en trop ! `{% empty %}` doit être à la même hauteur que `{% for %}`. ;)